Add cutting height for work areas to Husqvarna Automower (#32465)
home-assistant/home-assistant.github.io: source/_integrations/husqvarna_automower.markdown

### Number (if available)

#### Cutting height

> The integration will create a number entity for changing the cutting height of the mower. This entity is disabled by the default. You have to enable it manually, because it can't be detected with the API if the mower has the capabilities to change the cutting height remotely. Consult the handbook of the mower before you enable this function. Also it's possible, that you can use this entity only as sensor and not actively change the cutting height.

Reviewer: ```suggestion
The integration will create a number entity for changing the cutting height of the mower. This entity is disabled by default. You have to enable it manually. It can't be detected with the API if the mower has the capability to change the cutting height remotely. Before enabling this function, refer to the mower documentation. Depending on the mower, it is possible that you can use this entity only passively as a sensor and not actively to change the cutting height.
```
Tiny style tweak suggestions

> Possible values are 1 to 9.

Reviewer: ```suggestion
Possible values are 1 to 9.
```
What do the numbers indicate? 1 means I leave the grass high, 9 means as short as possible?

Author: 1 is short. I've detailed the text.

#### Cutting height for work areas

> The integration will create a number entity for changing the cutting height for each work area of the mower if your mower supports work areas. Possible values are 0% to 100%.

Reviewer: ```suggestion
The integration will create a number entity for changing the cutting height for each work area of the mower if your mower supports work areas. Possible values are 0% to 100%.
```
What does the percentage indicate? If I put 40, will it mow only 40% of the area? Or cut the grass down to 40% of it's current length? Or cut 40% of the current length and leave 60% standing?

Author: I've detailed the description.

Reviewer: Thank you for specifying.
